service.upnext/addon.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8' standalone='yes'?>
+<addon id="service.upnext" name="Up Next" provider-name="im85288" version="1.0.0">
+    <requires>
+        <import addon="xbmc.python" version="2.26.0"/>
+        <import addon="script.module.requests" version="2.9.1" />
+        <import addon="script.module.simplejson" version="3.3.0"/>
+    </requires>
+    <extension library="service.py" point="xbmc.service" start="login"/>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">Provides a netflix style pop up notification for watching the next episode with fancy/simple modes.</summary>
+        <description lang="en">Provides a netflix style pop up notification for watching the next episode with fancy/simple modes. A prompt to check if your still watching also occurs after a few automatically watched shows in a row.</description>
+        <platform>all</platform>
+        <language/>
```

Comment:
please drop language tag, not applied to script addons that do not provide content